mffpy/xml_files.py

```diff
@@ -289,6 +306,22 @@ def content(cls, fileDataType: str,  # type: ignore
             }
         }
 
+    def get_content(self):
```

Comment:
A short function comment would be good here and in `get_serializable_content` - the descriptor of content is kind of a vague one, so it would be good to explain what qualifies as content w/i the file when queried.